R/subscript-loc.R

```diff
@@ -483,19 +482,15 @@ cnd_body_vctrs_error_subscript_oob_location <- function(cnd, ...) {
   i <- i[!is.na(i)]
 
   oob <- i[i > cnd$size]
-  oob_enum <- enumerate(oob)
+  oob_enum <- vctrs_cli_vec(oob)
 
-  format_error_bullets(c(
-    x = glue::glue(ngettext(
-      length(oob),
-      "Location {oob_enum} doesn't exist.",
-      "Locations {oob_enum} don't exist."
-    )),
-    i = glue::glue(ngettext(
-      cnd$size,
-      "There are only {cnd$size} {elt[[1]]}.",
-      "There are only {cnd$size} {elt[[2]]}.",
-    ))
+  n_loc <- length(oob)
+  n <- cnd$size
+  elt <- cnd_subscript_element_cli(n, cnd)
+
+  # TODO: Switch to `format_inline()` and format bullets lazily through rlang
+  cli::format_error(c(
+    "i" = "{cli::qty(n_loc)} Location{?s} {oob_enum} do{?esn't/n't} exist, there {cli::qty(n)} {?is/are} only {elt}."
```

Comment:
```suggestion
    "i" = "{cli::qty(n_loc)} Location{?s} {oob_enum} do{?esn't/n't} exist; there {cli::qty(n)} {?is/are} only {elt}."
```

Reply:
As discussed on Slack, now uses two bullets as before, and the "past the end" phrasing.